include/couch_replicator_api_wrap.hrl

```diff
@@ -22,10 +21,9 @@
     timeout,            % milliseconds
     ibrowse_options = [],
     retries = 10,
-    wait = 250,         % milliseconds
+    wait = [250, 25],  % [normal error wait time, 429 backoff wait time]
```

Comment:
Would a tuple work better? ( {250, 25} ). Lists are usually for good for data that's growing or when have to match on head and tail...